distributions/aws/infra_configs/scripts/utils/__init__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import yaml
+import logging
+
+logger = logging.getLogger(__name__)
+
+CONFIG_FILE = "scripts/config.yaml"
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-route53-aliastarget.html
+CLOUDFRONT_HOSTED_ZONE_ID = "Z2FDTNDATAQYW2"
```

Comment:
Where are we getting this random string from?

Reply:
linked the documentation on line 10

For creating an alias record for other AWS resource, one needs hosted zone id and DNS name. Since CloudFront is a global service, there is only one hosted zone id

Reply:
expanded comment